frontend/src/queries/schema.ts

```diff
@@ -1070,6 +1070,8 @@ export type QueryStatus = {
     expiration_time?: string
     task_id?: string
     query_progress?: ClickhouseQueryProgress
+    /** @default high */
+    priority?: 'normal' | 'high'
```

Comment:
Hm, normal isn't really normal if it's the non-default value.
Overall `priority` as the name for this might be a bit misleading. I was looking for a place where we'd actually prioritize the execution in this PR, but I see that this only about labeling, and specifically chained vc. one-off requests. So maybe literally:

```suggestion
    /** @default false */
    is_part_of_chain?: boolean
```
(and the metric tag would also be `is_part_of_chain` or similar)?

Reply:
Valid concerns...

However, as soon as we use something else other than chain for any reason, this is off. I wouldn't bind that we want a bit more granular metrics to the exact implementation.

How about just `labels: {prio: high|med, .. potentially others in future}` to not confuse it with some processing priority?

Reply:
Or just `labels: ['chained']` and then use that for the metric accordingly. Will push a change.